typescript/virtual-pet/src/VirtualPet.ts

```diff
@@ -0,0 +1,41 @@
+export default class VirtualPet {
+  name: string
+  hunger: number
+  happiness: number
+
+  constructor(n: string) {
+    this.name = n
+    this.happiness = 50
+    this.hunger = 50
+  }
+  describe() {
```

Comment:
Class method return values can be annotated too:

```suggestion
  describe(): string {
```